index.js

```diff
@@ -2,36 +2,52 @@ const endpoint = process.env.coreAddr || 'localhost:50052'
 const mesg = require('mesg-js').application({ endpoint })
 const debug = require('debug')('marketplace')
 const error = require('debug')('marketplace:error')
-const { list } = require('./funcs/marketplace')
-const {
-  listenRequests,
-  response,
-  cacheResponse,
-} = require('./funcs/http-server')
+const executeTask = require('./utils/mesg.executeTask')(mesg)
+const listenEvent = require('./utils/mesg.listenEvent')(mesg)
+
+// update services cache on startup.
+cacheServicesResponse()
+
+// update services cache every 2 mins.
+setInterval(cacheServicesResponse, 60 * 1000 * 2)
```

Comment:
```suggestion
setInterval(cacheServicesResponse, 60 * 1000 * 60)
```

maybe 1 time per hour should be enough because of the event cache update..
maybe we could simply delete it

Reply:
done